docs/GettingStarted.md

```diff
@@ -339,23 +360,23 @@ If everything is set up correctly, you should see your new app running in your A
 
 `react-native run-android` is just one way to run your app - you can also run it directly from within Android Studio or [Nuclide](https://nuclide.io/).
 
-<block class="mac ios android" />
+<block class="mac ios tvos android" />
 
 ### Modifying your app
 
 Now that you have successfully run the app, let's modify it.
 
-<block class="mac ios" />
+<block class="mac ios tvos" />
 
 - Open `index.ios.js` in your text editor of choice and edit some lines.
-- Hit `Command⌘ + R` in your iOS Simulator to reload the app and see your change!
+- Hit `Command⌘ + R` in your iOS or Apple TV Simulator to reload the app and see your change!
```

Comment:
Since the same sentence will show up whether you're following the guide for iOS or Apple TV, I'd rather use "in your Simulator" (drop the "iOS or Apple TV") here.